src/IntegrationUtils.ts

```diff
@@ -52,6 +56,42 @@ export async function startIntegration(context: vscode.Uri): Promise<any> {
 						});
 					});
 					break;
+				case propertyIntegration:
+					let hasMoreProperties: boolean = true;
```

Comment:
what do you think to extract the propertyIntegration case inside a method, it will make it easier to read all the cases.